constant/item.js

```diff
@@ -1,97 +0,0 @@
-export const navItems = [
-    {
```

Comment:
@Seema111 I removed this static list of navigation menu items because we I think we should fetch these dynamically